vllm/v1/utils.py

```diff
@@ -407,7 +407,7 @@ def create_dp_placement_groups(
             # For now, each DP rank can only be assigned to one node
             # TODO(rui): support allocating a single DP rank
             # to multiple nodes
-            available_engine_count = node_resources["GPU"] // world_size
+            available_engine_count = int(node_resources["GPU"] // world_size)
```

Comment:
@chaunceyjiang I think this may be a bit clearer since `world_size` should always be an int.
```suggestion
            available_engine_count = int(node_resources["GPU"]) // world_size
```

Reply:
Done